src/components/Utilities.js

```diff
@@ -0,0 +1,88 @@
+// Utilities
+import React from 'React';
+import AsyncStorage from '@react-native-community/async-storage';
+
+export default class Utilities extends React.Component {
+
+    constructor() {
+        super(null);
+
+        this.get = function(key) { //returns string
+            try {
+                const value = AsyncStorage.getItem(key);
+                return value;
+            } catch (e) {
+                console.log(e);
+            }
+        }
+
+
+        this.store = function(key, value) { //takes in string
+            try {
+                AsyncStorage.setItem(key, value);
+            } catch (e) {
+                console.log(e);
+            }
+        }
+
+        this.store('graph', null);
+    }
+
+
+    async pullGraph() {
+        const stored = this.get('graph');
+
+        fetch('https://SASE-Labs-2020.github.io/assets/graph.json')
```

Comment:
Please see how we use `getData` [here](https://github.com/SASE-Labs-2020/gopherway/blob/master/src/components/GraphEdge.js#L35). Please add `getData` to `Utilities` and use `await getData`. 

This is needed because we want the asynchronous call to `fetch` to be resolved before the rest of the caller's code happens. See [this StackOverflow post](https://stackoverflow.com/a/48475017) for more details.